### PR TITLE
feat(dashboard): Add platform list

### PR DESCRIPTION
**Pull request by lynnagara**
screen shot 2018-04-30 at 10 51 03 am

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx [hunk 1]
```diff
@@ -12,6 +12,76 @@ import {Client} from 'app/api';
 import {update} from 'app/actionCreators/projects';
 import overflowEllipsis from 'app/styles/overflowEllipsis';
 
+class PlatformList extends React.Component {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx]: Should we move this into its own module?

File: src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx [hunk 2]
```diff
@@ -12,6 +12,76 @@ import {Client} from 'app/api';
...
+    return (
+      <StyledPlatformIconWrapper key={platform} className={platform}>
+        <StyledPlatformIcon className={`platformicon platformicon-${platform}`} />
+      </StyledPlatformIconWrapper>
+    );
+  }
+
+  getIcons(platforms) {
+    return (
+      <Flex direction="row-reverse" m={18}>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx]: is `18` on purpose here? what about `2`?

File: src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx [hunk 3]
```diff
@@ -12,6 +12,76 @@ import {Client} from 'app/api';
 import {update} from 'app/actionCreators/projects';
 import overflowEllipsis from 'app/styles/overflowEllipsis';
 
+class PlatformList extends React.Component {
+  static propTypes = {
+    platforms: PropTypes.arrayOf(PropTypes.string),
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx]: not required, would be v nice if we had a list of valid platform strings in `SentryTypes`

File: src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx [hunk 4]
```diff
@@ -12,6 +12,76 @@ import {Client} from 'app/api';
...
+      </Flex>
+    );
+  }
+  render() {
+    const {platforms} = this.props;
+
+    if (!platforms.length)
+      return (
+        <NoPlatforms align="center" p={2}>
+          No platforms yet
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx]: `t()`

File: tests/js/spec/views/organizationDashboard/projectCard.spec.jsx [hunk 5]
```diff
@@ -7,14 +7,17 @@ describe('ProjectCard', function() {
...
+        params={{orgId: 'org-slug'}}
+      />,
       TestStubs.routerContext()
     );
 
     projectMock = MockApiClient.addMockResponse({
       url: '/projects/org-slug/project-slug/',
       method: 'PUT',
-      data: TestStubs.Project({isBookmarked: false}),
+      data: TestStubs.Project({isBookmarked: false, platforms: ['javascript']}),
```
Comment (billyvg) [inline tests/js/spec/views/organizationDashboard/projectCard.spec.jsx]: can we add an assertion that `StyledPlatformIcon.platformicon.platformicon-javascript` exists?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx]: 👍

File: tests/js/spec/views/organizationDashboard/projectCard.spec.jsx (hunk 5 above)
Comment (lynnagara) [reply tests/js/spec/views/organizationDashboard/projectCard.spec.jsx]: 👍

File: src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx (hunk 3 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx]: Yeah, although I'm not sure what they are and if they are subject to change

Comment (billyvg) [reply src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx]: Yeah the reason why it'd be nice to have is that it's difficult to find the list.

Looks like this is the current set of platforms: https://sentry.io/settings/sentry/javascript/install/

File: src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx (hunk 2 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx]: Good catch, I think the margins are a couple of pixels off

**Pull request merged**